vizro-core/src/vizro/models/_components/form/dropdown.py

```diff
@@ -14,23 +13,14 @@
 from vizro.models.types import MultiValueType, OptionsDictType, OptionsType, SingleValueType
 
 
-def _get_list_of_labels(full_options: OptionsType) -> Union[list[StrictBool], list[float], list[str], list[date]]:
```

Comment:
FYI @nadijagraca removing this function is another simplification we can make now thanks to your earlier conversion of the `options` to `OptionsDictType` ⭐ 

I decided we shouldn't move the conversion into a pydantic validator though - it works well where it is I think.